src/content/insertable-streams/video-processing/js/webgpu-background-blur.js

```diff
@@ -193,6 +193,7 @@ const batch = [4, 4];
   constructor() {
     // All fields are initialized in init()
     /** @private {?OffscreenCanvas} canvas used to render video frame */
+    this.enableWorkaroundForGPUMemoryLeak = true;
```

Comment:
Avoid inserting this line between the js doc for `this.canvas_`. And probably, you should use underscore prefix for the private member.

Reply:
Done.